roles/openshift_control_plane/tasks/main.yml

```diff
@@ -180,23 +180,26 @@
   - fail:
       msg: Node start failed.
 
-- name: Verify that the control plane is running
-  command: >
-    curl -k {{ openshift.master.api_url }}/healthz/ready
-  args:
-    # Disables the following warning:
-    # Consider using get_url or uri module rather than running curl
-    warn: no
-  register: control_plane_health
-  until: control_plane_health.stdout == 'ok'
+- name: Wait for control plane pods to appear
+  oc_obj:
+    state: list
+    kind: pod
+    name: "master-{{ item }}-{{ openshift.node.nodename | lower }}"
+    namespace: kube-system
+  register: control_plane_pods
+  until:
+  - "'results' in control_plane_pods"
+  - "'results' in control_plane_pods.results"
+  - control_plane_pods.results.results | length > 0
   retries: 60
   delay: 5
-  changed_when: false
-  # Ignore errors so we can log troubleshooting info on failures.
-  ignore_errors: yes
+  with_items:
+  - "{{ 'etcd' if inventory_hostname in groups['oo_etcd_to_config'] else omit }}"
+  - api
+  - controllers
+  ignore_errors: true
 
-# Capture debug output here to simplify triage
-- when: control_plane_health.stdout != 'ok'
+- when: control_plane_pods | failed
```

Comment:
`is failed` per https://github.com/openshift/openshift-ansible/pull/8419/

Reply:
Fixed